goss.py

```diff
@@ -57,12 +61,17 @@
 
 
 # launch goss validate command on the file
-def check(module, test_file_path, output_format, goss_path):
-    cmd = ""
+def check(module, test_file_path, output_format, goss_path, vars_path):
+    cmd = "{0} -g {1}".format(goss_path, test_file_path)
+    # goss parent command flags
+    if vars_path is not None:
+        cmd += " --vars {0}".format(vars_path)
+
+    # validate sub-command flags
+    cmd += " validate"
```

Comment:
I did this because I thought using the long-form makes what's happening clearer to the reader of the code and the output, rather than requiring them to look up what the short-form means if they're unfamiliar.